test/manifests/maat_vtt_segmentTemplate.mpd

```diff
@@ -6,21 +6,33 @@
     <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1" lang="en">
       <Role schemeIdUri="urn:mpeg:dash:role:2011" value="main"></Role>
       <SegmentTemplate duration="95232" initialization="$RepresentationID$/init.m4f" media="$RepresentationID$/$Number$.m4f" startNumber="0" timescale="48000"></SegmentTemplate>
-      <Representation audioSamplingRate="48000" bandwidth="63000" codecs="mp4a.40.2" id="63000"></Representation>
-      <Representation audioSamplingRate="48000" bandwidth="125000" codecs="mp4a.40.2" id="125000"></Representation>
+      <Representation audioSamplingRate="48000" bandwidth="63000" codecs="mp4a.40.2" id="63000">
+        <SegmentTemplate></SegmentTemplate>
```

Comment:
all these `<segment*>` tags added to these mpd manifests shouldn't be necessary to add

Reply:
If you wanted to test your use case within an actual manifest, you could create a fresh one, and add that and the accompanying `json` to `index.test.js`.